### PR TITLE
Updated es.po and es.man

### PR DESCRIPTION
**Pull request by natastro**
Hi @suve 

Here is my pull request for #85 
`es.po` and `es.man` have both been updated. If you have any questions or concerns just let me know 😀

Thank you for letting me help with this!

### CODE REVIEW

File: lang/es.po [hunk 1]
```diff
@@ -40,7 +41,11 @@ msgstr "    Controla si la terminal debería de usar sequencias de escape\n"
        "    redirecciona a un archivo o a un pipe.\n"
 
 msgid "HELP_OPTION_DESCRIBE\n"
-msgstr "    Cuando enlistas paquetes, incluye descripción corta de los mismos.\n"
+msgstr "    Al listar paquetes, incluya los resúmenes de paquetes (descripciones breves).\n"
+
+msgid "HELP_OPTION_EVRA\n"
+msgstr "    Además de los nombres de los paquetes, imprima también su epoch:version-release.arch.\n"
```
Comment (suve) [inline lang/es.po]: Please wrap this to 80 characters per line - that being not 80 characters per line of the `.po` file, but 80 characters per line of the message (so from the opening `"` to the closing `\n"`).

Note: Wrapping is only needed for the `HELP_*` strings. `ERR_*` strings should be left single-line.

File: lang/es.po [hunk 2]
```diff
@@ -9,6 +9,7 @@ msgstr ""
 # Put something like "Latin translation by Jvlivs Gaivs\n" here.
 msgid "TRANSLATION_AUTHOR\n"
 msgstr "Traducción al Español por Mr. Outis\n"
+mgstr "Traducción al Español por Natalie Stroud\n"
```
Comment (suve) [inline lang/es.po]: 1. `mgstr` -> `msgstr` (the first `s` is missing)
2. Please keep the previous translator's name and add your name in an "X and Y" fashion.

File: lang/es.po [hunk 3]
```diff
@@ -40,7 +41,11 @@ msgstr "    Controla si la terminal debería de usar sequencias de escape\n"
        "    redirecciona a un archivo o a un pipe.\n"
 
 msgid "HELP_OPTION_DESCRIBE\n"
-msgstr "    Cuando enlistas paquetes, incluye descripción corta de los mismos.\n"
+msgstr "    Al listar paquetes, incluya los resúmenes de paquetes (descripciones breves).\n"
```
Comment (suve) [inline lang/es.po]: Wrap to 80 characters per line.

**Review by suve — CHANGES_REQUESTED** [review]
Thanks! Looks good overall, just a couple minor issues.

**Comment by natastro** [comment]
Hi @suve - I think I corrected what needed to be changed. If you still see an issue, just let me know! Thank you again!

File: lang/es.po [hunk 4]
```diff
@@ -9,6 +9,7 @@ msgstr ""
 # Put something like "Latin translation by Jvlivs Gaivs\n" here.
 msgid "TRANSLATION_AUTHOR\n"
 msgstr "Traducción al Español por Mr. Outis\n"
+msgstr "Traducción al Español por Natalie Stroud\n"
```
Comment (suve) [inline lang/es.po]: This creates two alternative translations for the string. What I'd like you do is something like:
```
msgtr "Traducción al Español por Mr. Outis y Natalie Stroud\n"
```
I don't know Spanish, so no idea if one should use `y` or `e` or something else here.

File: lang/es.po [hunk 5]
```diff
@@ -40,7 +41,13 @@ msgstr "    Controla si la terminal debería de usar sequencias de escape\n"
        "    redirecciona a un archivo o a un pipe.\n"
 
 msgid "HELP_OPTION_DESCRIBE\n"
-msgstr "    Cuando enlistas paquetes, incluye descripción corta de los mismos.\n"
+msgstr "    Al listar paquetes, incluya los resúmenes de paquetes (descripciones\n
+ breves).\n"
```
Comment (suve) [inline lang/es.po]:
```
msgstr "    Al listar paquetes, incluya los resúmenes de paquetes (descripciones\n"
       "    breves).\n"
```

File: lang/es.po [hunk 6]
```diff
@@ -40,7 +41,13 @@ msgstr "    Controla si la terminal debería de usar sequencias de escape\n"
...
+
+msgid "HELP_OPTION_EVRA\n"
+msgstr "    Además de los nombres de los paquetes, imprima también su\n
+ epoch:version-release.arch.\n"
+       "    El valor predeterminado es 'auto', lo que hace esto sólo para los paquetes que comparten un nombre.\n"
```
Comment (suve) [inline lang/es.po]:
```
msgstr "    Además de los nombres de los paquetes, imprima también su\n"
       "    epoch:version-release.arch. El valor predeterminado es 'auto', \n"
       "    lo que hace esto sólo para los paquetes que comparten un nombre.\n"
```

**Review by suve — CHANGES_REQUESTED** [review]

**Comment by natastro** [comment]
Hi @suve - so sorry about this! These should now be corrected but if you see anything else, feel free to let me know 😀

**Comment by suve** [comment]
Looks good now. Thanks!

**Pull request merged**